lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -635,42 +683,49 @@ fromShelleyPParams eraInfo pp = W.ProtocolParameters
 -- convert it into a percentage.
 --
 decentralizationLevelFromPParams
-    :: SLAPI.PParams era
+    :: HasField "_d" pparams SL.UnitInterval
```

Comment:
If we were to use `Cardano.Api` we wouldn't need to do all of this juggling. But looking into it, it seems a bit all-or-nothing — we can't start using it for a single query only. Unless we ask them to expose a bunch of internal stuff.